src/plugins/intel_gpu/src/runtime/format.cpp

```diff
@@ -136,6 +136,7 @@ static const std::map<format::type, format_traits> format_traits_map {
         FMT_TRAITS(os_i_osv16__ai8,                              1, 1, 0, 0, {0, 1},          "oi",     "oi",    {{1, 8}, {0, 16}},                 {{1, 8}, {0, 16}}),  // NOLINT
         FMT_TRAITS(os_i_osv8__ai8,                               1, 1, 0, 0, {0, 1},          "oi",     "oi",    {{1, 8}, {0, 8}},                  {{1, 8}, {0, 8}}),  // NOLINT
 
+        FMT_TRAITS(goix,                                         1, 1, 1, 1, {0, 1, 2, 3},       "goix",   "oix?????g", {},                         {}),  // NOLINT
```

Comment:
I have curiosity of this change. I dont find any code regarding goix. Only new type is added. Why this works?

Reply:
The error message of that issue was:
```
Error has occured for: convolution:aten::_convolution/GroupConvolution_4
Weights feature maps number(=1) is not equal to: input feature maps number(=192)
Weights/ifm mismatch
```
And the suspicious point was conversion of oiyx -> goix for filter_inst but there was no goix.
So just added the format and it worked.

Reply:
I cannot understand, why and how adding goix helped even though goix is not existing.

Reply:
The files before this PR are going to be used to explain:
When filter_inst is created:
![debug_1](https://github.com/user-attachments/assets/b0391a2a-9025-44d8-8ddf-1e38340ded64)
It refers to oiyx.
![debug_2](https://github.com/user-attachments/assets/12a5fa41-830b-42a9-9be3-0774c4007d5a)
![debug_3](https://github.com/user-attachments/assets/42cc3764-635c-491d-8226-82bc6d6b1100)
And wrong number is in group.
![debug_4](https://github.com/user-attachments/assets/a48efa20-b1af-4c65-8a30-a170fe8ae06a)
So goix in this PR is needed to provide right group value to existing codes.
What do you think?

Reply:
What was the app or test you met this error? What was the actual filter format used at the end in that case?

Reply:
It was found in static pipeline side on E2E:Dynamism. (E2E:Dynamism validates dynamic pipeline based on static pipeline.) And it is reproduced using benchmark_app.